karabo_data/geometry2/__init__.py

```diff
@@ -341,7 +341,14 @@ def position_all_modules(self, data, out=None):
         """Deprecated alias for :meth:`position_modules_fast`"""
         return self.position_modules_fast(data, out=out)
 
-    def plot_data_fast(self, data, axis_units='px', frontview=True):
+    def plot_data_fast(self,
+                       data,
+                       axis_units='px',
+                       frontview=True,
+                       ax=None,
+                       figsize=None,
+                       cbar=False,
```

Comment:
I'd be open to making this `True` by default. Thoughts - @tmichela or anyone?

I'd also prefer to spell out `colorbar` - it's both clearer and consistent with the naming in matplotlib and pandas.

Reply:
Ok, you're write!